hydro_deploy/core/src/localhost/mod.rs

```diff
@@ -156,14 +156,39 @@ impl LaunchedHost for LaunchedLocalhost {
         id: String,
         binary: Arc<(String, Vec<u8>, PathBuf)>,
         args: &[String],
+        perf: bool,
     ) -> Result<Arc<RwLock<dyn LaunchedBinary>>> {
-        let child = Command::new(&binary.2)
+        let mut command = if perf {
+            println!("Profiling binary with perf");
+            let mut tmp = Command::new("perf");
+            tmp.args([
+                "record",
+                "-F",
+                "5",
+                "--call-graph",
+                "dwarf,64000",
+                "-o",
+                (id.clone() + ".perf.data").as_str(),
```

Comment:
We should probably ask for a path in the `.perf(...)` method instead of picking one here?